gulpjs/examples/build/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "build",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Either you remove this script, or you add a npm test as a additional build step (but then don't `exit 1`).